Foundation/include/Poco/Dynamic/VarHolder.h

```diff
@@ -731,6 +767,264 @@ class VarHolderImpl: public VarHolder
 		return typeid(T);
 	}
 
+	void convert(Int8& val) const
+	{
+		if constexpr (std::is_enum<T>::value)
+		{
+			convertToSigned(std::underlying_type_t<T>(_val), val);
+		}
+		else {
```

Comment:
style (this one, and many others) - curly brace goes in the new line

Reply:
Ups, sorry.
Fixed